x/minfee/keeper/grpc_query.go

```diff
@@ -0,0 +1,31 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/celestiaorg/celestia-app/v4/x/minfee/types"
+)
+
+var _ types.QueryServer = &Keeper{}
+
+// NetworkMinGasPrice returns the network minimum gas price.
+func (k *Keeper) NetworkMinGasPrice(ctx context.Context, _ *types.QueryNetworkMinGasPrice) (*types.QueryNetworkMinGasPriceResponse, error) {
+	// delegate to the self managed params.
+	networkMinGasPrice := k.GetParams(sdk.UnwrapSDKContext(ctx)).NetworkMinGasPrice
+
+	// TODO: do we need to fallback to the subspace method for v3 and below?
```

Comment:
Yes would be great but it won't allow us to ever delete params. Maybe with the passthrough command it is fine.
Would be amazing if nova would eventually do that for queries too